### PR TITLE
Remove timeouts

### PR DESCRIPTION
**Pull request by bthaile**
removed code no longer needed.

We are moving away form font awesome, there were many setTimeout violations when rocket-loader was trying to pull in font awesome.

Also removed a setTimeout having to do with setting market not loading, missed those on previous refactor.

https://github.com/AugurProject/augur/issues/1479

### CODE REVIEW

File: src/modules/common/components/input-dropdown/input-dropdown.jsx [hunk 1]
```diff
@@ -125,12 +125,11 @@ class InputDropdown extends Component {
...
         </select>
-        <i
-          className={classNames(Styles.InputDropdown__icon, "fa", {
-            "fa-angle-down": !this.state.showList,
-            "fa-angle-up": this.state.showList
-          })}
-        />
+        <span className={Styles.InputDropdown__icon}>
+          {!this.state.showList
+            ? ChevronDown({ stroke: "#FFFFFF" })
```
Comment (phoebemirman) [inline src/modules/common/components/input-dropdown/input-dropdown.jsx]: can you use the ChevronFlip component for this ? Then you won't have to reuse code.

Comment (bthaile) [reply src/modules/common/components/input-dropdown/input-dropdown.jsx]: updated two places to use ChevronFlip.

File: src/modules/forking/components/forking-alert/forking-alert.styles.less [hunk 2]
```diff
@@ -49,6 +49,15 @@
 
 .ForkingAlert__arrow {
   margin-left: 1em;
+
+  svg {
+    height: 1em;
+    margin-right: 0.5em;
+
+    polyline {
+      stroke: @color-white;
```
Comment (phoebemirman) [inline src/modules/forking/components/forking-alert/forking-alert.styles.less]: do you still need this here and in the other spot?

File: src/modules/common/components/input-dropdown/input-dropdown.styles.less [hunk 3]
```diff
@@ -73,6 +73,14 @@
 
 .InputDropdown__icon {
   align-self: center;
```
Comment (phoebemirman) [inline src/modules/common/components/input-dropdown/input-dropdown.styles.less]: this should be flex-end so the arrow is aligned

Comment (bthaile) [reply src/modules/common/components/input-dropdown/input-dropdown.styles.less]: made the correction.

**Comment by bthaile** [comment]
Fixed search functionality.

**Pull request merged**